### ISSUE TITLE
Will Nutrients searches be added to the new FoodData Central API soon?

### ISSUE THREAD

**Issue by yoganinja** [USDA/USDA-APIs#65]
I was just getting started with the Food Composition Database with Nutrients searches being a key part of my app. The new API does not have an endpoint for that, unless I'm missing something. I even tried doing my nutrient search to the old API and using the returned NDBno to get the details from the new API in the fdcId field, but it seems all the food Ids have changed as well. I understand the API is just weeks old, so I'm hoping it will be expanded soon...?

**Comment by littlebunch** [comment]
I agree the lack of a nutrient reporting capability is a deficiency that needs to be addressed and it is on the developers todo list as well as the ID issues.
Gary
